pkg/sql/logictest/testdata/logic_test/builtin_function

```diff
@@ -308,6 +308,31 @@ abc abc abc
 query error substring\(\): negative byte subarray length -1 not allowed
 SELECT substring('11100011'::bytea, 4, -1)
 
+query TTTTT
```

Comment:
protip for next time, you can do:
```
query T
SELECT
   encode(substring(decode(str, 'hex'), start), 'hex')
FROM ( VALUES
  ('ff0001', 1),
  ('ff0001', 2),
) tbl(str, start)
````

so you only need one T :P

Reply:
i can feel my brain growing as we speak!